virttest/virsh.py

```diff
@@ -983,7 +984,7 @@ def driver(**dargs):
     """
     # libvirt schme composed of driver + command
     # ref: http://libvirt.org/uri.html
-    scheme = urllib.parse.urlsplit(canonical_uri(**dargs))[0]
+    scheme = urllib.parse.urlsplit(canonical_uri(decode_to_text(**dargs)))[0]
```

Comment:
I don't see necessity to decode_to_text(**dargs) here

Reply:
@chunfuwen I tried to make sure the type of parameter is text